src/ui/setup/store.ts

```diff
@@ -42,6 +43,7 @@ let reducers = {
   layout: layoutReducer,
   messages: messagesReducer,
   protocolMessages: protocolMessages,
+  newSources: sources,
```

Comment:
:speech_balloon: heh, I _think_ this is meant to be "this is the new sources reducer, different than the existing `sources` reducer", but it's amusing that we have the type name `newSource` from the backend API :)